Directory.Packages.props

```diff
@@ -30,6 +29,10 @@
     <PackageVersion Include="Microsoft.VisualBasic" Version="$(MicrosoftVisualBasicVersion)" />
     <PackageVersion Include="Microsoft.VisualStudio.Composition" Version="$(MicrosoftVisualStudioCompositionVersion)" />
     <PackageVersion Include="NETStandard.Library" Version="$(NETStandardLibraryVersion)" />
+    <!-- 
+      Pinning transitive NuGet dependencies to stop Microsoft.CodeAnalysis.Analyzer.Testing from 
+      bringing in an older NuGet which still has .NET Standard 1.3 dependencies.
+    -->
```

Comment:
Is this needed to deal with the MSBuildWorkspace + NuGet.Frameworks incompatibilities? Because those should be addressed by the build host work.